documentation/book/proc-accessing-kafka-using-ingress.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// assembly-configuring-kafka-listeners.adoc
+
+[id='proc-accessing-kafka-using-ingress-{context}']
+= Accessing Kafka using {KubernetesName} ingress
+
```

Comment:
Shall we add an intro to the procedure? 'In this procedure, ...'

Reply:
Done.